src/Graphqelm/Generator/Decoder.elm

```diff
@@ -56,7 +56,7 @@ generateDecoder apiSubmodule (Type.TypeReference referrableType isNullable) =
             ]
 
         Type.InputObjectRef _ ->
-            Debug.crash "Input objects are only for input not responses, shouldn't need decoder."
+            Debug.todo "Input objects are only for input not responses, shouldn't need decoder."
```

Comment:
`Debug.todo` is the only equivalent of `Debug.crash`, but the docs state "Note: This is not available with elm make --optimize or packages." so I guess we'll need a different solution in these cases.